src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -100,6 +100,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Interactive.Async" Version="3.2.0" />
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
```

Comment:
nit: System.Memory brings System.Buffers transitively, it it worth adding it explicitly?

Reply:
that was a hangover from when I wasn't using System.Memory for this; no longer needed, removing